Day 2- updates based on dry run (#50)
AlexsLemonade/training-modules: scRNA-seq/00-scRNA-seq_introduction.md

- Sequencing is not bidirectional so data will likely have more intense 3' bias.  
- Coverage of these technologies is generally not as deep.  

## Resources:

> - Work-on-your-own [scRNA-seq training course](https://hemberg-lab.github.io/scRNA.seq.course/index.html)

Reviewer: ```suggestion
- Hemberg Lab [scRNA-seq training course](https://hemberg-lab.github.io/scRNA.seq.course/index.html)
```

> - GUI to do scRNA-seq processing things for you on [ASAP website][https://asap.epfl.ch/]

Reviewer: ```suggestion
- [ASAP: Automated Single-cell Analysis Pipeline](https://asap.epfl.ch/) is a web server that allows you to process scRNA-seq data. ([Gardeux et al. _Bioinformatics._ 2017.](https://doi.org/10.1093/bioinformatics/btx337 ))
```

> - UMI [concepts article](https://cgatoxford.wordpress.com/2015/08/14/unique-molecular-identifiers-the-problem-the-solution-and-the-proof/)

Reviewer: ```suggestion
- Smith. [_Unique Molecular Identifiers – the problem, the solution and the proof_](https://cgatoxford.wordpress.com/2015/08/14/unique-molecular-identifiers-the-problem-the-solution-and-the-proof/) provides an excellent background on UMIs.
```
